src/globus_cli/commands/timer/_common.py

```diff
@@ -66,7 +66,7 @@ def render(self, value: dict[str, t.Any]) -> str:
             start = value.get("start")
             if start:
                 start = formatters.Date.render(formatters.Date.parse(start))
-            end = value.get("end", {})
+            end = value.get("end") or {}
```

Comment:
Example Response payload:
```json
{
  "body": {...trunc...}
  "inactive_reason": null,
  "job_id": "e1981add-30e5-4c4f-8457-000235cbdba5",
  "last_ran_at": null,
  "name": "CLI Created Timer [2024-05-13T14:38:16.056771]",
  "next_run": "2024-05-13T19:38:16+00:00",
  "number_of_errors": 0,
  "number_of_runs": 0,
  "schedule": {
    "end": null,
    "interval_seconds": 60,
    "start": "2024-05-13T19:38:16+00:00",
    "type": "recurring"
  },
  "status": "loaded",
  "submitted_at": "2024-05-13T19:38:17+00:00"
}
```

Notably "$.schedule.end" is explicitly set to null, not omitted.

As a result `data["schedule"].get("end", {})` will return `None` not `{}`